eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -1,6 +1,7 @@
 parameters:
   Artifacts: []
   ServiceDirectory: not-specified
+  Coverage: ''
```

Comment:
Shouldn't this be set to `enabled` by default instead of empty?

Reply:
I guess you set this to enabled in the stage template so it doesn't matter what it is set to here.